io_scene_nif/modules/object/object_export.py

```diff
@@ -265,7 +265,8 @@ def export_node(self, b_obj, parent_block, node_name):
         if b_obj:
             # export animation
             if b_obj_ipo:
-				# no anim support here for now!
+                # no anim support here for now!
+                print(b_obj,"has anims")
```

Comment:
NifLog info or debug, unless this was just for dev work

Reply:
Yeah that was just temporary. Will be removed.